cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -12,10 +12,28 @@ class HttpCache
   # A map from script name to script level URL pattern.
   CACHED_SCRIPTS_MAP = {
       # We specify the "special case" routes here; standard routes are handled by the loop below.
-      'hourofcode' => '/hoc/*'
+      'hourofcode' => '/hoc/*',
+      'flappy' => '/flappy/*'
   }
-  %w(starwars starwarsblocks mc frozen gumball).each do |script_name|
-    CACHED_SCRIPTS_MAP[script_name] = "/s/#{script_name}/stage/1/puzzle/*"
+
+  # Cached scripts
+  %w(
+    starwars
+    starwarsblocks
+    mc
+    frozen
+    gumball
+    20-hour
+    course1
+    course2
+    course3
+    course4
+    playlab
+    artist
+    infinity
+    iceage
+  ).each do |script_name|
```

Comment:
The list at L107 marks certain paths in Pegasus/code.org as cached and language-keyed (as opposed to the Pegasus default, which is cached as English-only). That list might need some updating eventually (depending on whether we add new pages/paths to Code.org site), but would be unrelated to this Code Studio-specific change.